### PR TITLE
Prepare data for model

### PR DESCRIPTION
**Pull request by AbelTu0620**
Add model running part into Django structure and be tested.

### CODE REVIEW

**Review by alisonrclarke — COMMENTED** [review]
Looking good! Could you move `prepare_test_data()` and `excel_to_matrix` to tests.py? I don't think we need them in the tasks, do we?

File: manyfews/calculations/GenerateRiverFlows.py [hunk 1]
```diff
@@ -417,4 +382,185 @@ def GenerateRiverFlows(t0, gefsData, F0, parametersFilePath):
...
+    storageLevel = list(initialConditions.values("storage_level"))
+
+    slowFlowRateList = []
+    fastFlowRateList = []
+    storageLevelList = []
+
+    for i in range(len(initialConditions)):
+        slowFlowRateList.append(slowFlowRate[i]["slow_flow_rate"])
+        fastFlowRateList.append(fastFlowRate[i]["fast_flow_rate"])
+        storageLevelList.append(storageLevel[i]["storage_level"])
```
[the remark this answers is not in the document]
Comment (AbelTu0620) [reply manyfews/calculations/GenerateRiverFlows.py:413]: The answer is the same as above.

File: manyfews/calculations/GenerateRiverFlows.py [hunk 2]
```diff
@@ -1,23 +1,10 @@
-"""
```
[the remark this answers is not in the document]
Comment (AbelTu0620) [reply manyfews/calculations/GenerateRiverFlows.py:1]: OK, accepted and modified.

File: manyfews/calculations/tasks.py [hunk 3]
```diff
@@ -48,3 +57,51 @@ def prepareZentra():
...
+        os.path.join((os.path.split(os.path.realpath(__file__))[0]), "../../")
+    )
+
+    dataFileDirPath = os.path.join(projectPath, "Data")
+    parametersFilePath = os.path.join(
+        dataFileDirPath, "RainfallRunoffModelParameters.csv"
+    )
+
+    gefsData = prepareGEFSdata(date=dataDate, location=dataLocation)
+    intialConditionData = prepareInitialCondition(date=dataDate, location=dataLocation)
```
[the remark this answers is not in the document]
Comment (AbelTu0620) [reply manyfews/calculations/tasks.py:85]: OK, accepted and modified.

File: manyfews/calculations/GenerateRiverFlows.py [hunk 4]
```diff
@@ -417,4 +382,185 @@ def GenerateRiverFlows(t0, gefsData, F0, parametersFilePath):
...
+    """
+    # prepare testing GEFS data for model.
+    gefs = NoaaForecast.objects.filter(date=date).filter(location=location)
+
+    RH = list(gefs.values("relative_humidity"))
+    minTemperature = list(gefs.values("min_temperature"))
+    maxTemperature = list(gefs.values("max_temperature"))
+    uWind = list(gefs.values("wind_u"))
+    vWind = list(gefs.values("wind_v"))
+    precipitation = list(gefs.values("precipitation"))
```
[the remark this answers is not in the document]
Comment (AbelTu0620) [reply manyfews/calculations/GenerateRiverFlows.py]: accepted and modified.

File: manyfews/calculations/GenerateRiverFlows.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (AbelTu0620) [reply manyfews/calculations/GenerateRiverFlows.py]: accept and modified.

**Review by alisonrclarke — APPROVED** [review]

**Pull request merged**